### PR TITLE
Initial revision

### PR DESCRIPTION
**Pull request by crit**
Pragmatic update to get our internal systems up and running.

Will tag with version 1 as soon as comparable testing suite to Prismic's company supported libraries is completed.

A full refactor may happen as version 2.

### CODE REVIEW

**Comment by polds** [comment]
LGTM. Looks like those Panics are fixed for the most part then?

File: test/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// test integration with Prismic.io. todo @remove https://github.com/prismicio/javascript-kit/blob/master/test/test.js
+package test
+
+import (
+	"github.com/kyani-inc/goprismic"
+)
+
+const (
+	repo = "https://micro.cdn.prismic.io/api"
+	token = "MC5VcXBHWHdFQUFONDZrbWp4.77-9cDx6C3lgJu-_vXZafO-_vXPvv73vv73vv70777-9Ju-_ve-_vSLvv73vv73vv73vv70O77-977-9Me-_vQ"
```
Comment (RossValler) [inline test/integration_test.go]: Should this be an env var instead?

Comment (crit) [inline test/integration_test.go]: Good idea but this reflects the integration tests used by Prismic's libraries: 

example: https://github.com/prismicio/javascript-kit/blob/master/test/test.js

I'm following their lead for now.

Comment (RossValler) [inline test/integration_test.go]: Okay cool :)

**Comment by crit** [comment]
@polds Yeah, removes the only `panic()` call.

**Pull request merged**